python/manylinux1/scripts/build_boost.sh

```diff
@@ -16,12 +16,12 @@
 # specific language governing permissions and limitations
 # under the License.
 
-BOOST_VERSION=1.68.0
+BOOST_VERSION=1.73.0
 BOOST_VERSION_UNDERSCORE=${BOOST_VERSION//\./_}
 NCORES=$(($(grep -c ^processor /proc/cpuinfo) + 1))
 
-curl -sL https://sourceforge.net/projects/boost/files/boost/${BOOST_VERSION}/boost_${BOOST_VERSION_UNDERSCORE}.tar.gz -o /boost_${BOOST_VERSION_UNDERSCORE}.tar.gz
-tar xf boost_${BOOST_VERSION_UNDERSCORE}.tar.gz
+curl -sL https://dl.bintray.com/boostorg/release/${BOOST_VERSION}/source/boost_${BOOST_VERSION_UNDERSCORE}.tar.bz2 -o /boost_${BOOST_VERSION_UNDERSCORE}.tar.bz2
```

Comment:
Could you keep sourceforge.net URL?
Because dl.bintray.com/boostorg/ will reach limitation constantly. See also: https://github.com/boostorg/boost/issues/299

Reply:
Ok, I'll do. I changed to Bintray because the bandwidth to SF is bad here :-/

Reply:
Well, I don't know about Bintray limitations, but SF looks incredibly crappy and limited:
https://dev.azure.com/ursa-labs/crossbow/_build/results?buildId=13366&view=logs&j=0da5d1d9-276d-5173-c4c4-9d4d4ed14fdb&t=d9b15392-e4ce-5e4c-0c8c-b69645229181&l=3325

Reply:
The Bintray limitation is bandwidth: https://github.com/boostorg/boost/issues/299#issuecomment-605462600

> The problem is not the reliability of the servers. The problem is that we have a large, but limited amount of (donated) bandwidth (about 30 TB/month). Someone could set up a set of mirrors, but who is going to provide (pay for) the service?

It may be better that we mirror Boost at our Bintray and use our mirror.